src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/AllServicesOrgRoleTest.java

```diff
@@ -627,6 +627,38 @@ void shouldDeleteOrgRequestedRoleForHearing(String roleName, String roleCategory
     }
 
 
+    @ParameterizedTest
+    @CsvSource({
+        "hearing-manager,SYSTEM,PRIVATELAW,Benefit",
+        "hearing-viewer,SYSTEM,PRIVATELAW,Benefit"
```

Comment:
For read-ability the test Jurisdiction and Case Type should probably match up: e.g. 
* PRIVATELAW => PRLAPPS
* SSCS => Benefit
i.e. the only reason in the test to expect failure should be the mis-matched process and jurisdiction: not the mis-matched case-type  (I know this is not a rule enforced here but it is in many other rule/config scenarios).

Reply:
Resolved in 14c0ad52b5d834d1ff2b0bc8584bdb5ad5cef610